bentoml/_internal/bento/gen.py

```diff
@@ -21,11 +6,13 @@
 import logging
 from typing import TYPE_CHECKING
 
+import fs
 import attr
 from jinja2 import Environment
 from jinja2.loaders import FileSystemLoader
 
-from ..utils import bentoml_cattr
+from ..utils import bentoml_cattr as cattr
```

Comment:
let's not do this, it's very confusing - we should make it very clear it is our cattr instance

Reply:
note that there are still places where we use cattr directly.